src/msg/async/AsyncConnection.cc

```diff
@@ -82,7 +82,7 @@ class C_clean_handler : public EventCallback {
  public:
   explicit C_clean_handler(AsyncConnectionRef c): conn(c) {}
   void do_request(int id) {
-    conn->cleanup_handler();
+    conn->cleanup();
```

Comment:
On Tue, Jul 12, 2016 at 5:37 PM, Kefu Chai <notifications@github.com> wrote:

> In src/msg/async/AsyncConnection.cc
> <https://github.com/ceph/ceph/pull/10090#discussion_r70406327>:
>
> > @@ -82,7 +82,7 @@ class C_clean_handler : public EventCallback {
> >   public:
> >    explicit C_clean_handler(AsyncConnectionRef c): conn(c) {}
> >    void do_request(int id) {
> > -    conn->cleanup_handler();
> > +    conn->cleanup();
>
> It needs all fd used locally which not like kernel socket shared by all
> cores.
>
> so you mean, we need to pin the fds to its own EventCenter to prepare for
> the DPDK? but how is this mapped to DPDK?
>
dpdk stack will be the driver of eventcenter, like EventEpoll now. It will
be EventDPDK. So pin eventcenter means pin dpdk stack instance too.


> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ceph/ceph/pull/10090/files/63d3fbd01f5f15e06477b891c22fd9963af2599b..0ef50041a48f7efaf583ce1bbfb9e56138b3593a#r70406327>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AA72YQgG0YRmmYyFMDjbnnT-QCLBuDhjks5qU2BcgaJpZM4JDDWp>
> .
>



-- 

Best Regards,

Wheat